content/blog/2022/01/2022-01-14-happy-new-year.adoc

```diff
@@ -0,0 +1,254 @@
+The pull requests included improvements to Jenkins core, Jenkins plugins, Jenkins infrastructure, and Jenkins documentation.
+
+Key results included:
+
+* link:/blog/2021/10/31/hacktoberfest-results-2021/#translations-and-artwork[Translations and artwork], including improvements in French, Spanish, and Russian language translations and two new Jenkins images
+* link:/blog/2021/10/31/hacktoberfest-results-2021/#plugin-docs-migration-to-github[Plugin documentation migration to GitHub] for 29 different plugins, with a https://www.youtube.com/watch?v=NU2g373wHNo&t=2331s[tutorial video] to help new contributors
+* link:/blog/2021/10/31/hacktoberfest-results-2021/#implementing-content-security-policy[Security improvements in content security policy] from six contributors moving Javascript into dedicated files
+* link:/blog/2021/10/31/hacktoberfest-results-2021/#jenkins-architecture-diagrams[Architecture diagrams] providing both a dataflow view and a high level overview of Jenkins
+* Five part video series on link:https://youtu.be/Fev8KfFsPZE["Modernizing Jenkins Plugins"]
+
+=== She Code Africa Contributhon
+
+image:/images/post-images/2021-04-contributhon-participants/shecodeafrica-participants-opengraph.png[She Code Africa Contributhon 2021, role=center, link=/blog/2021/04/07/contributhon-participants/]
+
+The Jenkins project was a mentoring organization in the first ever link:https://sites.google.com/shecodeafrica.org/contributhon[She Code Africa Contributhon].
+The She Code Africa Contributhon pays African women to work with open source organizations on selected projects with dedicated mentors.
+This program aims to create a more diverse, inclusive, and innovative culture within the African open source ecosystem by matching African women in technology with sponsor and mentor open source organizations.
+
+The five women met with mentors twice weekly for a month.
+They built Jenkins core and Jenkins plugins and submitted pull requests to improve Jenkins plugin online help and documentation.
+The 5 mentees on the Jenkins project joined the project from Nigeria, Kenya, and Rwanda.
+
+More details are available in the link:/blog/2021/04/07/contributhon-participants/[She Code Africa blogpost].
+
+== Sponsors
+
+The Jenkins project is truly fortunate to have so many sponsors contributing to its success.
+We thank our sponsors sincerely and look forward to their continuing support.
+The Jenkins project depends on sponsor organizations for its infrastructure, tools, and funding.
+
+Many contributors are employed by sponsor companies like CloudBees and Red Hat.
+Jenkins distribution, build, test, and deployment infrastructure is provided by sponsors like JFrog, Amazon Web Services, Oregon State University Open Source Lab, Oracle, and Digital Ocean.
+Jenkins development is tracked with tools provided by sponsors like GitHub, JFrog, Atlassian, the Linux Foundation, and Netlify.
```

Comment:
Is it worth adding 1password here? we haven't really started using it, but they did give us a free team.

Reply:
Added.  Thanks for the reminder